components/org.wso2.carbon.identity.api.server.authenticators/org.wso2.carbon.identity.api.server.authenticators.common/src/main/java/org/wso2/carbon/identity/api/server/authenticators/common/AuthenticatorsServiceHolder.java

```diff
@@ -77,4 +79,24 @@ public void setIdentityProviderManager(IdentityProviderManager identityProviderM
 
         AuthenticatorsServiceHolder.getInstance().identityProviderManager = identityProviderManager;
     }
+
+    /**
+     * Get ApplicationAuthenticatorService osgi service.
+     *
+     * @return ApplicationAuthenticatorService
+     */
+    public ApplicationAuthenticatorService getApplicationCommonService() {
```

Comment:
Why this method name is 'getApplicationCommonService' ? This is basically of retrieving "ApplicationAuthenticationService"